coffeelint.json

```diff
@@ -1,103 +0,0 @@
-{
```

Comment:
yeah this should probably be part of this ticket, I've done some of the files already.